packages/components/inputs/select-utils/src/create-select-styles.ts

```diff
@@ -58,6 +92,15 @@ type TState = {
   isSelected?: boolean;
 };
 
+type TDesignTokenName = keyof typeof designTokens;
+
+const getHorizontalConstraintValue = (value?: string | number) => {
```

Comment:
I think the type of this value should be `TProps['minMenuWidth'] | TProps['maxMenuWidth']`, right?